src/configuration/config_crypter.go

```diff
@@ -114,7 +128,7 @@ func encryptConfig(key, value string) string {
 	return encryptSimple(keyByte, value)
 }
 
-func decryptConfig(key, value string) string {
+func decryptConfig(key, value string) (string, error) {
```

Comment:
May not be necessary?

Reply:
These are not passed up, but they should be. I'll fix it.

Reply:
Initially I missed the bubbling of the return.